torch/onnx/utils.py

```diff
@@ -334,47 +334,37 @@ def _trace_and_get_graph_from_model(model, args):
     return trace_graph, torch_out
 
 
-def _model_to_graph(model, args, verbose=False,
-                    input_names=None, output_names=None,
-                    operator_export_type=OperatorExportTypes.ONNX,
-                    example_outputs=None, propagate=False,
-                    _retain_param_name=False, do_constant_folding=True,
-                    _disable_torch_constant_prop=False, fixed_batch_size=False):
-    from torch.onnx.symbolic_helper import _export_onnx_opset_version
-    # Special case for common case of passing a single Tensor
-    if isinstance(args, torch.Tensor):
-        args = (args, )
-
-    if isinstance(example_outputs, torch.Tensor):
-        example_outputs = [example_outputs]
-
+def _model_to_jit_graph(model, args, example_outputs, retain_param_name, enable_jit_freeze_module):
```

Comment:
`example_outputs` seems not needed in this method.

Reply:
How about for assertions?

Reply:
can probably move assertions outside this method? seems only needed later